.circleci/config.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.1
+    steps:
+      - checkout
+      - run:
+          name: 'Creating Virtualenv'
+          command: virtualenv venv
+      - restore_cache:
+          key: v2-requirements-{{ checksum "requirements.txt" }}
```

Comment:
It v2 because caches are immutable and I updated the contents of what gets cached from v1 - so we can't use v1